deploy/cloudformation/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "cloudformation",
+  "version": "1.0.0",
+  "description": "## Template and parameters",
+  "main": "create-stacks.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
You have tests, though. Why not to run them via `npm test`?

Reply:
It's a shorthand for remote e2e test, not a test for package itself.